docs/DeveloperGuide.adoc

```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |edit the detail of a person
+ |easily make change to their detail when they updated their contact
+
+|`* * *` |user |delete a person |remove entries that I no longer need
+
+|`* * *` |user |create a class |group and manage students who are taking the same class
+
+|`* * *` |user |add persons to a class |
+
+|`* * *` |user |see all persons in a class |see the list of all students taking a class
+
+|`* *` |user |add appointment with a person to my schedule |be reminded of the appointment
+
+|`* *` |user |delete appointment from my schedule |remove appointment I no longer need to be reminded
+
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*` |user with many persons in address book |tag a person |mark their contact with a detail
```

Comment:
teacher with many students in address book | tag a student | mark their contact with details

Reply:
no need for 'with many students in address book'. User should be able to tag regardless of number of contacts